src/models/index.js

```diff
@@ -25,5 +25,8 @@ const sequelize = new Sequelize(
 
 
 module.exports = {
-    sequelize
+    sequelize,
+    itemSale,
```

Comment:
please fix the typos in the exports names
itemSale must be itemSaleModel
 
also, we might not use the models directly in the exports